cluster/stone-prd-rh01/tenants/konflux-o11y-tenant/integration_test_scenario.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: project-controller-enterprise-contract
+  namespace: konflux-o11y-tenant
+spec:
+  application: project-controller
+  contexts:
+    - description: Application testing
+      name: application
+  resolverRef:
+    params:
+      - name: url
+        value: 'https://github.com/redhat-appstudio/build-definitions'
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipelines/enterprise-contract-redhat-no-hermetic.yaml
```

Comment:
@ernesgonzalez33 Yes, it's going to be deployed. I was following the document and according to it `redhat-no-hermetic` was used. What should be used here? Previously as well I used the same.

Reply:
I think you could add already the `POLICY_CONFIGURATION` parameter and point it to the `rhtap-releng-tenant/app-interface-standard`. Because that's the one that probably will have to be used in the release

Reply:
Updated. Please check.